clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/AutoElasticsearchDocumentMapperSpec.scala

```diff
@@ -17,14 +17,11 @@ class AutoElasticsearchDocumentMapperSpec extends FlatSpec with Matchers {
     DocumentMock
   ](() => expectedId)
 
-  it should "id" in {
-    mapper.id(ModelMockKey(12345L, "key")) should be("12345.key")
-  }
-
   it should "empty" in {
     mapper.empty(ModelMockKey(12345L, "key")) should be(
       DocumentMock(
         clioId = expectedId,
+        entityId = "12345.key",
```

Comment:
Since this cross-references the data on lines 31 and 32, the code should use constants to make it clear that where the matched value is coming from. 

The use of "." here should also use a constant which would be defined in `AutoElasticsearchDocumentMapper` (or wherever it's used).